include/caf/events/force_power_down_event.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#ifndef _FORCE_POWER_DOWN_EVENT_H_
+#define _FORCE_POWER_DOWN_EVENT_H_
+
+#include <event_manager.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Simple event to keep force the quick power down
```

Comment:
Simple event to force a quick power down

Reply:
fixed